components/docs/plugins/createApp.tsx

```diff
@@ -9,9 +9,9 @@ import OptionsList from "./optionsList";
 const PluginDocsCreateApp = ({ plugin }: { plugin: string }) => {
   if (!plugin) return null;
 
-  const cmdName = plugin.toLowerCase();
-  const appId = `my_${cmdName}_app`;
   const pluginData = PluginData.find(({ name }) => name === plugin);
+  const cmdName = pluginData.slug;
+  const appId = `my_${cmdName.replace("-", "_")}_app`;
```

Comment:
This is our first plugin with a multi-word name!  To fit the casing conventions of our plugins/docs consistent, I added this.  Slugs and commands will use hyphens, object ids will remain snake case.